runtime-modules/project-token/src/lib.rs

```diff
@@ -942,15 +1026,36 @@ impl<T: Trait> Module<T> {
 
     pub(crate) fn validate_transfers(
         token_id: T::TokenId,
-        transfers: TransfersOf<T>,
+        transfers: TransfersWithVestingOf<T>,
         transfer_policy: &TransferPolicyOf<T>,
-    ) -> Result<ValidatedTransfers<T>, DispatchError> {
+        is_issuer: bool,
+    ) -> Result<ValidatedTransfersOf<T>, DispatchError> {
         let result = transfers
+            .0
             .into_iter()
             .map(|(dst, payment)| {
-                Self::validate_destination(token_id, dst, transfer_policy).map(|res| (res, payment))
+                let dst_acc_data =
```

Comment:
Referring to the previous comment, I would say:
1. use `ensure_account_data_exist` here,
2. if necessary add the vesting schedule params after making sure that the extra VS can be added
3. return tagged `account_id` with `Existing/NonExistng` + payment params

Reply:
Addressed in https://github.com/Joystream/joystream/pull/3719/commits/f3f6c258ae8c707a878ed568bef3c133b213b866, hope I got that right